src/desktop/src/ui/views/onboarding/Login.js

```diff
@@ -301,7 +301,6 @@ const mapStateToProps = (state) => ({
     additionalAccountName: state.accounts.accountInfoDuringSetup.name,
     ui: state.ui,
     currency: state.settings.currency,
-    onboarding: state.ui.onboarding,
```

Comment:
What are we removing this? 

/cc @rihardsgravis

Reply:
It's not used